keyVaultClientSideToCustomerProvidedServerSide/setup/config.py

```diff
@@ -0,0 +1,23 @@
+from azure.storage.blob import CustomerProvidedEncryptionKey
+
+KEYVAULT_URL = ["your_keyvault_url"]
+CLIENT_ID = ["your_client_id"]
+CLIENT_SECRET = ["your_client_id"]
+TENANT_ID = ["your_tenant_id"]
+connection_str = ["your_azure_storage_account_connection_str"]
+
+# items to make encryption scope
+SUB_ID = ["your_subscription_id"]
+RESOURCE_GROUP = ["your_resource_group_name"]
+STORAGE_ACCOUNT = ["your_storage_account_name"]
+
+# if using customer provided key, set values here
+customer_key = CustomerProvidedEncryptionKey(key_value="MDEyMzQ1NjcwMTIzNDU2NzAxMjM0NTY3MDEyMzQ1Njc=",
```

Comment:
if these are fake values, I'd add that these are fake insecure values - otherwise maybe remove them?

Reply:
or moved to configuration file.